packages/ui/src/components/va-data-table/VaDataTable.vue

```diff
@@ -428,133 +432,140 @@ export default defineComponent({
 @import "./variables";
 // The calculated variables are taken from a respective element's `style` attribute. See the `useStyleable` hook
 
-.va-data-table {
-  width: 100%;
-  cursor: default;
+.va-data-table__wrapper {
```

Comment:
That's pretty weird nesting bem-wise

Reply:
Fixed: removed the nesting of css-selectors.